### DISCUSSION TITLE
R2S作为旁路由，流量经过openclash无论国内外都跑不满，百兆只能跑五十兆

### DISCUSSION THREAD

**Discussion by nggezi** [vernesong/OpenClash#3779] [Q&A]
clash接管dns，刷了openwrt官方固件只装openclash效果相同，不知道该发哪种日志就先不发了，如果需要看日志可以告诉我在哪找出来

**Comment by nggezi** [comment]
cpu占用和内存占用都不到一半，应该不是性能瓶颈

**Comment by nggezi** [comment]
查到了，是设备的数据发到软路由，软路由不知道为什么复制了一份同样的数据，两份一起发向外网，导致设备只能占一半带宽

**Reply by othelloYu** [reply]
> 查了一下，是设备数据发到软路由，软路由不知道为什么复制使用同样的数据，两份一起发向外网，导致设备只能占一半带宽

请问这个问题您是如何排查出来的呢？我使用别的插件 科学感觉比openclash快了一倍不止，不知道是不是也有这种问题？

**Reply by nggezi** [reply]
软路由只连一台设备，跑单线程测速，设备的速度是clash面板连接板块显示的速度的正好一半，但是openclash和openwrt系统的流量是相同的，所以不确定是否是openclash还是固件本身的问问题，切换其他插件确实会好转，但是现在过去了几个月已经慢慢恢复了，姑且当做是固件和插件没磨合好或者设备性能问题吧

**Comment by Wanglaoba666** [comment]
大佬怎么解决

**Reply by nggezi** [reply]
没解决，凑合着用了，好像固件磨合一下就缓和了，现在大概能跑80%
